lib/active_public_resources/drivers/youtube.rb

```diff
@@ -128,29 +128,21 @@ def parse_video(item)
         video.title          = snippet['title']
         video.description    = snippet['description']
         video.thumbnail_url  = snippet['thumbnails']['default']['url']
-        video.url            = "https://www.youtube.com/watch?v=#{video_id}&feature=youtube_gdata_player"
-        video.embed_url      = "https://www.youtube.com/embed/#{video_id}?feature=oembed&rel=0"
```

Comment:
When a YouTube video gets returned to Canvas from the YouTube edu_apps tool, was it using this `embed_url`, or the `url` property?

I'm guessing it was using the `url` property (since we are deleting the `embed_url` here), but wanted to make sure :smile:

Reply:
It was using both.  Right below this in the "returnTypes" it showed which URL it was using for which type.  But since we're only using the embed_url, it seemed appropriate to just give a URL :)

Reply:
I mean... technically Canvas was only using URL, because the returnType below only gave it "url", it just chose whether embed_url or url was the url it would send.